### PR TITLE
View Potential Candidate Information

### PR DESCRIPTION
**Pull request by vohuunhat25**
This feature handle View Potential Candidate Information
Please review.

### CODE REVIEW

File: src/main/java/com/demoipm/controller/PotentialCandidatesController.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import com.demoipm.dto.CandidateDto;
+import com.demoipm.entities.Candidate;
+import com.demoipm.service.CandidateService;
+import com.demoipm.service.PotentialCandidateService;
+
+@Controller
+public class PotentialCandidatesController {
+	
+	@Autowired
+	PotentialCandidateService potentialCandidateService;
```
Comment (MaHai1993) [inline src/main/java/com/demoipm/controller/PotentialCandidatesController.java:20]: properties MUST be private.

File: src/main/java/com/demoipm/service/impl/PotentialCandidateImpl.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import com.demoipm.entities.Candidate;
+import com.demoipm.service.PotentialCandidateService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+@Service
+@Transactional
+public class PotentialCandidateImpl implements PotentialCandidateService{
+
+	@Autowired
+	PotentialCandidateDao potentialCandidateDao;
```
Comment (jason103nguyen) [inline src/main/java/com/demoipm/service/impl/PotentialCandidateImpl.java:21]: Please private property of class

File: src/main/java/com/demoipm/service/impl/PotentialCandidateImpl.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	
+	private ObjectMapper mapper;
+	
+	@Override
+	public void createPotentialCandidate(CandidateDto candidateDto) {
```
Comment (jason103nguyen) [inline src/main/java/com/demoipm/service/impl/PotentialCandidateImpl.java:26]: Please add comment for method and check similar case

**Review by jason103nguyen — CHANGES_REQUESTED** [review]
Please fix comments

**Pull request merged**